code/modules/mob/living/simple_animal/bot_swarm/swarmer.dm

```diff
@@ -163,8 +163,27 @@
 	S.DisIntegrate(src)
 	deactivate(S, 0)
 
-/obj/machinery/particle_accelerator/control_box/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
-	S.DisIntegrate(src)
+/obj/machinery/particle_accelerator/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	S << "<span class='warning'>Disrupting the power grid would bring no benefit to us. Aborting.</span>"
+
+/obj/structure/particle_accelerator/fuel_chamber/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	S << "<span class='warning'>Disrupting the power grid would bring no benefit to us. Aborting.</span>"
+
+/obj/structure/particle_accelerator/particle_emitter/center/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
```

Comment:
@tigercat2000 I know but when I tried it In the manner that wasn't fucking stupid it didn't work which made me yell for a few minutes